packages/orchestration/src/exos/ibc-packet.js

```diff
@@ -88,6 +89,10 @@ export const prepareIBCTransferSender = (zone, { watch, makeIBCReplyKit }) => {
     }),
     {
       public: {
+        /**
+         * @param {Vow<IBCEvent<'acknowledgementPacket' | 'timeoutPacket'>>} match
```

Comment:
I think this needs to be:
```suggestion
         * @param {Vow<IBCEvent<'acknowledgementPacket'> | IBCEvent<'timeoutPacket'>>} match
```

Reply:
Yes, thank you. `IBCEvent<'acknowledgementPacket' | 'timeoutPacket'>` is not equivalent to your suggestion - it seems to narrow to fields common to both 'acknowledgementPacket' and 'timeoutPacket'.